components/dialog/index.css

```diff
@@ -359,3 +359,16 @@ only screen and (max-device-height: 350px) {
     transform: none;
   }
 }
+
+.spectrum-Dialog--error {
+
+  .spectrum-Dialog-typeIcon {
```

Comment:
I try to keep the scope as small as possible to avoid messing up the scenario that has button in the upper right corner. Please see https://github.com/adobe/spectrum-css/pull/520/files#r375056334 also.

Reply:
I see, but note that this example is the only place `.spectrum-Dialog-typeIcon` is used, so I think we're good.